include/boost/type_index/stl_type_index.hpp

```diff
@@ -39,7 +39,13 @@
 #include <boost/type_traits/remove_cv.hpp>
 #include <boost/type_traits/remove_reference.hpp>
 
-#include <boost/container_hash/hash.hpp>
+#if (defined(_MSC_VER) && _MSC_VER > 1600) \
+    || (defined(__GNUC__) && __GNUC__ == 4 && __GNUC_MINOR__ > 5 && defined(__GXX_EXPERIMENTAL_CXX0X__)) \
+    || (defined(__GNUC__) && __GNUC__ > 4 && __cplusplus >= 201103)
+#   define BOOST_CLASSINFO_STD_TYPE_INDEX_HAS_HASH_CODE
```

Comment:
Could you please change the macro name to start with `BOOST_TYPE_INDEX_`? I'm afraid that some day there will be a Boost.Classinfo library and a collision could happen.

Reply:
Yes, sure. I was unsure which prefix to use and copied one from the other define.

Reply:
Oh... I have to fix that. Thanks fot the heads up!